bittensor/utils/networking.py

```diff
@@ -199,3 +199,19 @@ def upnpc_create_port_map(port: int):
 
     except Exception as e:
         raise UPNPCException(e) from e
+
+def get_formatted_ws_endpoint_url(endpoint_url: str) -> str:
+    """
+    Returns a formatted websocket endpoint url.
+    Note: The port (or lack thereof) is left unchanged
```

Comment:
Given this note It sounds that it would be nice to comment examples of the cases you are formatting

Reply:
Given that you have tests.... you can just delegate this responsibility of the cases from this code covers to the test file